src/style/global.scss

```diff
@@ -56,6 +56,10 @@ html {
   @include iui-anchor;
 }
 
+.iui-visually-hidden {
```

Comment:
I am not sure whether it is ok to have this class globally.

Reply:
hm..
Initially, Id thought to have some file like `utils` which you would include if you need such classes.
But as this is a single class, maybe it can stay :/